8.x/core-concepts/annotations.md

```diff
@@ -2,4 +2,423 @@
 
 [[toc]]
 
-Should be filled.
\ No newline at end of file
+Annotations are the tags declaring metadata for the program source code. They provide additional information about the program to the compiler but are not part of the program itself. These annotations do not affect the execution of the compiled program, but add some new traits to the classes, methods and properties.*<sup>[1](#1)</sup>*
+
+#### Example in java language
+
+```java{1-19}
+class Flower {
+  public void displayInfo() {
+    System.out.println("I am a flower.");
+  }
+}
+
+class Rose extends Flower {
+  @Override
+  public void displayInfo() {
+    System.out.println("I am a rose.");
+  }
+}
+
+class Main {
+  public static void main(String[] args) {
+    Rose r1 = new Rose();
+    r1.displayInfo();
+  }
+}
+```
+**OUTPUT**
+
+```bash{1}
+I am a rose
+```
+
+In the above example, both the superclass and subclass include the method displayInfo(). However the method of the subclass is called during the program execution.
+
+## Annotations in php language
+
+Annotations are called `DocBlock` in the php language. A DocBlock is a piece of documentation in the source code that informs about a certain class, method or other structural elements.*<sup>[2](#2)</sup>*
+
+#### Example in php language
+
```

Comment:
There is not a fixed syntax for PHPDoc annotations. This means that anything that comes after the '@name' symbol is interpreted in various ways depending on the type of annotation being used. For instance, the '@var' annotation follows the syntax of '@var {type} {description}', whereas the '@param' annotation follows a slightly different syntax of '@param {type}{$name} {description}'.

Reply:
Ok. Thanks to share.